code/game/objects/items/flamethrower.dm

```diff
@@ -214,13 +214,22 @@
 	//TODO: DEFERRED Consider checking to make sure tank pressure is high enough before doing this...
 	//Transfer 5% of current tank air contents to turf
 	var/datum/gas_mixture/air_transfer = ptank.air_contents.remove_ratio(release_amount)
-	if(air_transfer.gases[/datum/gas/plasma])
-		air_transfer.gases[/datum/gas/plasma][MOLES] *= 5
+	if(air_transfer.gases[/datum/gas/plasma] || air_transfer.gases[/datum/gas/tritium] || air_transfer.gases[/datum/gas/hydrogen])
+		if(air_transfer.gases[/datum/gas/plasma])
+			air_transfer.gases[/datum/gas/plasma][MOLES] *= 5
+		if(air_transfer.gases[/datum/gas/tritium])
+			air_transfer.gases[/datum/gas/tritium][MOLES] *= 6.5
+		if(air_transfer.gases[/datum/gas/hydrogen])
+			air_transfer.gases[/datum/gas/hydrogen][MOLES] *= 7
+	else
+		air_transfer.temperature += 15000
```

Comment:
you prob shouldn't add a way to dupe hydrogen and trit, the hotspot **should** take care of heating the contents up enough, unless you were running into something particular I'd remove it.

Reply:
and why plasma is *=5 ? i'll remove the trit and h2 ones, i can't remove air_transfer.temperature because all other gases are not heated up in there

Reply:
the gases should get heated up when released into the atmosphere and exposed to a hotspot, I hope?

The plasma is *= 5 because someone went `not enough plasma is coming out, can't make tanks bigger, lets just make more plasma!` and it stuck

Reply:
i tested this with the gas_y and nitrogen, but the heating is negligible and it cools off immediately, maybe is because of the very small amount of gas released?

that sound stupid lol

Reply:
it really is

Reply:
is gas y flammable? you may wanna try mixing it with something that is, plasma maybe? (I know I know)